service/core/kafka/server/src/main/scala/com/lightbend/lagom/internal/broker/kafka/TopicProducerActor.scala

```diff
@@ -39,9 +39,45 @@ import akka.stream.scaladsl.Zip
 import java.net.URI
 
 import akka.kafka.ProducerMessage
+import akka.stream.scaladsl.RestartSource
+import com.lightbend.lagom.internal.broker.kafka.TopicProducerActor.Start
 
-private[lagom] class TaggedOffsetProducerActor[Message](
+private[lagom] object TopicProducerActor {
```

Comment:
Moved `object` to the top of the file but, more importantly, `s/TaggedOffsetProducerActor/TopicProducerActor`